job-dsls/src/main/resources/job-scripts/prod_kogito_create_images_templates_and_sources.jenkinsfile

```diff
@@ -0,0 +1,46 @@
+pipeline {
```

Comment:
How about renaming the jenkinsfile to **prod_kogito_create_images_sources.jenkinsfile**